modules/fedimint-ln-client/Cargo.toml

```diff
@@ -32,7 +32,7 @@ bitcoin = { workspace = true }
 clap = { workspace = true, optional = true }
 erased-serde = { workspace = true }
 fedimint-api-client = { workspace = true }
-fedimint-client = { workspace = true }
+fedimint-client = { workspace = true, package = "fedimint-client-module" }
```

Comment:
This way client module can depends on `fedimint-client-module` but sees it as `fedimint-client`, to avoid going around and renaming all the `use` statement. For time being.